support-frontend/assets/pages/supporter-plus-landing/components/contributionsPriceCards.tsx

```diff
@@ -0,0 +1,114 @@
+import { css } from '@emotion/react';
+import { from, headline, space, textSans } from '@guardian/source-foundations';
+import { Button } from '@guardian/source-react-components';
+import { useNavigate } from 'react-router-dom';
+import { OtherAmount } from 'components/otherAmount/otherAmount';
+import { PriceCards } from 'components/priceCards/priceCards';
+import { PriceCardsContainer } from 'components/priceCards/priceCardsContainer';
+import { countryGroups } from 'helpers/internationalisation/countryGroup';
+import { resetValidation } from 'helpers/redux/checkout/checkoutActions';
+import {
+	useContributionsDispatch,
+	useContributionsSelector,
+} from 'helpers/redux/storeHooks';
+import { navigateWithPageView } from 'helpers/tracking/ophan';
+
+const titleAndButtonContainer = css`
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	margin: 6px 0 ${space[3]}px;
+	${from.desktop} {
+		margin-bottom: 0;
+	}
+`;
+
+const title = css`
+	${headline.xsmall({ fontWeight: 'bold' })}
+	${from.tablet} {
+		font-size: 28px;
```

Comment:
Maybe worth a `${headline.small({ fontWeight: 'bold' })}` instead?

Reply:
the initial font size declaration `xsmall` is correct 24px ... as for the font-size changing from tablet breakpoints I don't think a proper way to do this has been decided while using source, with that in mind simply changing the font-size is the smallest change to the css we can make in order to make this change and avoid re-declaring duplicate css propertties